### DISCUSSION TITLE
Wrong assertion of columns in CSV upload leads to wrong customer IBAN

### DISCUSSION THREAD

**Discussion by fabianhugo** [pretix/pretix#4458] [Q&A]
Dear community, 
pretix is an amazing piece of software! 
by user mistake, i uploaded all ~600 payments with the wrong colum set as customer IBAN. And now all payments have the organizer IBAN instead, which leads to problems when using automated refund payments. I was able to update the saved csv assertion table by uploading a completely different csv file. My question is if there is any way to update all the IBANs to the correct ones.

**Comment by raphaelm** [comment]
Unfortunately there is no ready-made way to update these in bulk :(

**Reply by fabianhugo** [reply]
Okay, thank you :)
